Source/ACE/Entity/Player.cs

```diff
@@ -2516,6 +2521,49 @@ public ActionChain CreateMoveToChain(ObjectGuid target, float distance)
             return moveToChain;
         }
 
+        public ActionChain CreateCollideWithChain(ObjectGuid target)
```

Comment:
its also a change from MoveToObject to MoveToPosition in movementTypes.

the code works, it might be more convoluted but it does function and until physics are in its fine for now.